omero/sysadmins/unix/install-web/web-deployment.rst

```diff
@@ -72,7 +69,6 @@ Walkthroughs
     walkthrough/omeroweb-install-centos8-ice3.6
     walkthrough/omeroweb-install-ubuntu1804-ice3.6
     walkthrough/omeroweb-install-ubuntu2004-ice3.6
-    walkthrough/omeroweb-install-debian9-ice3.6
```

Comment:
Should this be redirected to the debian10 walkthrough similarly to the server walkthrough?

Otherwise, the redirect logic works for me

Reply:
I made the changes locally but I forgot to commit and push.
Done now